modules/openshift-developer-cli-configuring-debugging-parameters.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/debugging-applications-in-odo.adoc
+
+[id="configuring-debugging-parameters_{context}"]
+
+= Configuring debugging parameters
+
+.Procedure
+
+* To set a remote port on which the debugging agent should run, use:
++
+----
+$ odo config set DebugPort 9292
+----
+
+[NOTE]
+====
+You must re-deploy your component for this value to be reflected on the component.
+====
+
+* To set a local port that would be port forwarded, run:
++
+----
+$ odo debug port-forward --local-port 9292
+----
+
+[NOTE]
+====
+The local port value does not persist, provide it every time you need to change the port.
```

Comment:
This isn't a full sentence. Either use a colon instead of the comma, or break it into two sentences.

```
 The local port value does not persist; you must provide it every time you need to change the port.
or 
 The local port value does not persist. You must provide it every time you need to change the port.
```